src/compute/src/render.rs

```diff
@@ -824,27 +842,43 @@ where
         self.scope.clone().region_named("Main Body", |region| {
             let depends = body.depends();
             self.enter_region(region, Some(&depends))
-                .render_letfree_plan(body)
+                .render_letfree_plan(object_id, body)
                 .leave_region()
         })
     }
 
     /// Renders a plan to a differential dataflow, producing the collection of results.
     ///
     /// The plan must _not_ contain any `Let` or `LetRec` nodes.
-    fn render_letfree_plan(&mut self, plan: FlatPlan) -> CollectionBundle<G> {
-        let (mut nodes, root_id, topological_order) = plan.destruct();
+    fn render_letfree_plan(&mut self, object_id: GlobalId, plan: FlatPlan) -> CollectionBundle<G> {
+        let (mut steps, root_id, topological_order) = plan.destruct();
 
         // Rendered collections by their `LirId`.
         let mut collections = BTreeMap::new();
 
-        for id in topological_order {
-            let node = nodes.remove(&id).unwrap();
-            let mut bundle = self.render_plan_node(node, &collections);
-
-            self.log_operator_hydration(&mut bundle, id);
+        for lir_id in topological_order {
+            let step = steps.remove(&lir_id).unwrap();
+
+            // TODO(mgree) need ExprHumanizer in DataflowDescription to get nice column names
+            // (ActiveComputeState doesn't have a catalog reference)
+            let operator = step.node.humanize(&DummyHumanizer);
+
+            let operator_id_start = self.scope.peek_identifier();
+            let mut bundle = self.render_plan_node(step.node, &collections);
+            let operator_id_end = self.scope.peek_identifier();
+            let operator_span = (operator_id_start, operator_id_end);
```

Comment:
Nit: We should only do this work if logging is enabled. A nit because logging is enabled by default.

Edit: I take that back, we're only logging non-transient dataflows, so we should avoid the cost of humanizing the dataflow if we don't need the operator string.

Reply:
Yep, done. Kind of awkward because of the bracketing, though. 😔